completions/bash/podman

```diff
@@ -2013,6 +2055,168 @@ _podman_logout() {
      _complete_ "$options_with_args" "$boolean_options"
 }
 
+
+_podman_pod_create() {
+  local options_with_args="
+      --cgroup-parent
+      --cgroup-to-ctr
+      --podidfile
+      --label-file
+      --label
+      -l
+      --name
+  "
+
+  local boolean_options="
+  "
+  _complete_ "$options_with_args" "$boolean_options"
+}
+
+# _podman_pod_inspect() {
+#   _podman_container_run
+# }
+#
+# _podman_pod_kill() {
+#   _podman_container_run
+# }
+
+__podman_pod_ps() {
+  local options_with_args="
+   -f
+   --filter
+   --format
+   --sort
+  "
+
+  local boolean_options="
+   --no-trunc
+   -q
+   --quiet
+   --cgroup
+   --labels
+  "
+  _complete_ "$options_with_args" "$boolean_options"
+}
+
+_podman_pod_ls() {
+  __podman_pod_ps
+}
+
+_podman_pod_list() {
+  __podman_pod_ps
+}
+
+_podman_pod_ps() {
+  __podman_pod_ps
+}
+
+# _podman_pod_pause() {
+#   _podman_container_run
+# }
+#
+# _podman_pod_restart() {
+#   _podman_container_run
+# }
+
+_podman_pod_rm() {
+  local options_with_args="
+  "
+
+  local boolean_options="
+    -a
+	--all
+    -f
+	--force
+	-rmctr
+  "
+  _complete_ "$options_with_args" "$boolean_options"
+    case "$cur" in
+        -*)
+            COMPREPLY=($(compgen -W "$boolean_options $options_with_args" -- "$cur"))
+            ;;
+        *)
+            __podman_complete_pod_names
+            ;;
+    esac
+}
+
+_podman_pod_start() {
+  local options_with_args="
+  "
+
+  local boolean_options="
+  "
+  _complete_ "$options_with_args" "$boolean_options"
+    case "$cur" in
+        -*)
+            COMPREPLY=($(compgen -W "$boolean_options $options_with_args" -- "$cur"))
+            ;;
+        *)
+            __podman_complete_pod_names
+            ;;
+    esac
+}
+
+# _podman_pod_stats() {
+#   _podman_container_run
+# }
+
+# _podman_pod_stop() {
+#   _podman_container_run
+# }
+
+# _podman_pod_top() {
+#   _podman_container_run
+# }
+
+# _podman_pod_unpause() {
+#   _podman_container_run
+# }
+#
+# _podman_pod_wait() {
+#   _podman_container_run
+# }
+
+
+_podman_pod() {
+    local boolean_options="
+    --help
+    -h
+    "
+    subcommands="
+     create
+     inspect
+     kill
+     ls
```

Comment:
I think you should shift `ls` down into `local aliases` and bring `ps` up into `subcommands`, because `ps` is our canonical name for the command.